### PR TITLE
[bitnami/grafana-operator]: Use Bitnami Grafana image by default

### LINKED ISSUE #17576: [bitnami/grafana-operator] New major version doesn't use Bitnami images
### Name and Version

bitnami/grafana-operator 3.X.X

### What architecture are you using?

None

### What steps will reproduce the bug?

Install the Bitnami Grafana Operator chart with any `3.x.x` version.

### Are you using any custom parameters or values?

No

### What is the expected behavior?

Grafana container should be launched using Bitnami Grafana image: https://hub.docker.com/r/bitnami/grafana

### What do you see instead?

Grafana container using this image instead: https://hub.docker.com/r/grafana/grafana

```console
$ kubectl get deploy -n monitoring -o yaml grafana-operator-grafana-deployment
(...)
    image: docker.io/grafana/grafana:9.1.6
    imagePullPolicy: IfNotPresent
(...)
```

### Additional information

This happened after the changes introduced at https://github.com/bitnami/charts/pull/17152 to adopt the new Grafana Operator major version (`5.0.0`) which is a massive re-write of the operator logic and the CRD(s). Check the link below to obtain more information:

- https://github.com/grafana-operator/grafana-operator/blob/master/docs/blog/v5-intro.md 

As you can see in the PR (see **deployment.yaml**), the `--grafana-image` flags were removed (the operator binary doesn't support them any longer) and now it's expected that you indicate the Grafana image to use when creating the Grafana object, see the CRD:

- https://github.com/grafana-operator/grafana-operator/blob/master/config/grafana.integreatly.org_grafanas.yaml#L1489-L1496

However, the default Grafana instance included in the chart doesn't set this property, see:

- https://github.com/bitnami/charts/blob/main/bitnami/grafana-operator/templates/grafana.yaml

### LINKED ISSUE #17807: [bitnami/grafana-operator]: ability to overwrite Grafana image
### Name and Version

bitnami/grafana-operator 3.0.3

### What is the problem this feature will solve?

Ability to run grafana-operator in regulated environments that do not allow the use of Docker Hub images.

### What is the feature you are proposing to solve the problem?

In the `values.yaml` of the Bitnami grafana-operator chart, one should be able to overwrite the _Grafana_ image. For this to work, the `image:` block (as shown [here](https://grafana-operator.github.io/grafana-operator/docs/examples/grafana_deployment/readme/)) needs to be added to the CR defined in [templates/grafana.yaml](https://github.com/bitnami/charts/blob/main/bitnami/grafana-operator/templates/grafana.yaml).

### What alternatives have you considered?

_No response_

### PR DESCRIPTION
**Pull request by juan131**
### Description of the change

This PR ensures the Bitnami Grafana Operator chart uses the [Bitnami Grafana image](https://hub.docker.com/r/bitnami/grafana) in the default's Grafana instance.

### Benefits

Bitnami charts to use exclusively Bitnami container images by default.

### Possible drawbacks

None

### Applicable issues

- fixes #17576
- fixes #17807

### Additional information

N/A

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/grafana-operator/README.md [hunk 1]
```diff
@@ -72,6 +72,7 @@ For more information, refer to the [documentation on the differences between the
 | `global.imageRegistry`    | Global Docker image registry                    | `""`  |
 | `global.imagePullSecrets` | Global Docker registry secret names as an array | `[]`  |
 
+
```
Comment (Mauraza) [inline bitnami/grafana-operator/README.md:75]: Could you remove this line?
```suggestion
```

File: bitnami/grafana-operator/README.md [hunk 2]
```diff
@@ -85,6 +86,7 @@ For more information, refer to the [documentation on the differences between the
 | `commonAnnotations` | Common Annotations which are applied to every ressource deployed                                          | `{}`            |
 | `clusterDomain`     | Kubernetes cluster domain name                                                                            | `cluster.local` |
 
+
```
Comment (Mauraza) [inline bitnami/grafana-operator/README.md:89]:
```suggestion
```

File: bitnami/grafana-operator/README.md [hunk 3]
```diff
@@ -184,78 +186,86 @@ For more information, refer to the [documentation on the differences between the
 | `operator.startupProbe.failureThreshold`                     | Failure threshold for startupProbe                                                                                                        | `3`                        |
 | `operator.startupProbe.successThreshold`                     | Success threshold for startupProbe                                                                                                        | `1`                        |
 
+
```
Comment (Mauraza) [inline bitnami/grafana-operator/README.md:189]:
```suggestion
```

File: bitnami/grafana-operator/README.md [hunk 4]
```diff
@@ -184,78 +186,86 @@ For more information, refer to the [documentation on the differences between the
...
+| `grafana.readinessProbe.failureThreshold`                   | Failure threshold for readinessProbe                                                                    | `6`                      |
+| `grafana.readinessProbe.successThreshold`                   | Success threshold for readinessProbe                                                                    | `1`                      |
+| `grafana.updateStrategy`                                    | Set up update strategy for Grafana installation.                                                        | `{}`                     |
+| `grafana.extraVolumes`                                      | Optionally specify extra list of additional volumes for the grafana pod(s)                              | `[]`                     |
+| `grafana.extraVolumeMounts`                                 | Optionally specify extra list of additional volumeMounts for the grafana container(s)                   | `[]`                     |
+| `grafana.sidecars`                                          | Add additional sidecar containers to the grafana pod(s)                                                 | `[]`                     |
+
```
Comment (Mauraza) [inline bitnami/grafana-operator/README.md:268]:
```suggestion
```

File: bitnami/grafana-operator/README.md (hunk 2 above)
Comment (juan131) [reply bitnami/grafana-operator/README.md]: Done at https://github.com/bitnami/charts/pull/17730/commits/6da6a77eacc9f1cc1f829647495b4b8233661814

File: bitnami/grafana-operator/README.md (hunk 4 above)
Comment (juan131) [reply bitnami/grafana-operator/README.md:266]: Done at https://github.com/bitnami/charts/pull/17730/commits/6da6a77eacc9f1cc1f829647495b4b8233661814

**Pull request closed without merging**